sources/packages/web/src/views/aest/student/StudentAccountApplicationsApproval.vue

```diff
@@ -24,7 +24,7 @@
                   color="primary"
                   @click="createStudentAccount"
                   :disabled="notAllowed"
-                  >Create account for student</v-btn
```

Comment:
This is the text on the button, I couldn't see an AC to revise button text and would request to have the business add the AC. Let me know if I am missing something.

Reply:
It was the same "Create account for student" text so I figured the first AC applied to both places that had that phrase on the page.

Reply:
Absolutely makes sense. But the AC doesn't seem to be specific on mentioning both places and requires to be updated.

Reply:
Got confirmation from the business on the same. Approving the PR.